tracing-flame/README.md

```diff
@@ -0,0 +1,128 @@
+# tracing-flame
+
+A Tracing [Layer][`FlameLayer`] for generating a folded stack trace for generating flamegraphs
+and flamecharts with [`inferno`]
```

Comment:
this isn't _strictly_ `inferno`-specific; it emits the `perf` output format, right? so technically any flamegraph tool that expects that can use this crate's output.

i'm not sure if this is worth stating _here_ though, for the sake of brevity...

Reply:
I don't think its the same format that perf emits, my understanding is that with perf normally you take the output and pass it through another script that folds the stack traces, and then you pass the folded representation into the flamegraph generating scripts. With `tracing-flame` we skip straight to emitting a folded representation. I think it might be cool to investigate if we can emit perf's exact format so we can be compatible with more tools, but I think it would be the same as writing a script to fake a bunch of stack traces that reproduce the folded repr, so for now I don't think we should bother with it.

Reply:
Yeah, this is the folded output produced by `flamegraph`, `inferno`, and some newer eBPF perf tracing tools. `perf`'s output (by which you're probably referring to `perf script`'s output) is probably not something we want to mirror. The collapsed output is decently standard, enough so that eBPF also produces the same thing.